app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
```

Comment:
@eshadatta It would be DRYer if you just rewrote the above function:

```ruby
def render_field_item(doc)
  doc[:document][doc[:field]].join(", ").truncate(450).html_safe
end
```

There is no use-case for a field with more than 450 chars so I don't see this as an issue.